js/components/select.tsx

```diff
@@ -181,12 +181,12 @@ class Select extends React.Component<IItemsComponentProps, IDropDownState> {
             this.setState({ isOpen });
             
             const select = this.comboBoxRef.current;
-            const mainScroller = (select).closest('.main-scroller');
+            const mainScroller = $(select).closest('.main-scroller');
 
             // innerHeight - offsetTop gives us the space available underneath the select box
             const documentSpaceUnderDropdown = window.innerHeight - select.offsetTop;
             // every bit we've scrolled down give us more space under the dropdown
-            const viewSpaceUnderDropdown = documentSpaceUnderDropdown + mainScroller.scrollTop;
+            const viewSpaceUnderDropdown = documentSpaceUnderDropdown + mainScroller.scrollTop();
```

Comment:
Don't think this is a function?

Reply:
weirdly, it is: https://api.jquery.com/scrolltop/

Reply:
ah, because it's a jQuery wrapped element. OK.